### PR TITLE
qty issue for puca theme

### PR DESCRIPTION
**Pull request by poojarazor**

### CODE REVIEW

File: templates/rzp-pdp-checkout-btn.php [hunk 1]
```diff
@@ -22,15 +22,23 @@ class="button alt single_add_to_cart_button"
 </div>
 
 <script type="">
-  var quantity = document.getElementsByClassName("qty")[0].value;
+  let i = 0;
+  while (typeof quantity === 'undefined') {
```
Comment (varun-315) [inline templates/rzp-pdp-checkout-btn.php:26]: why is a `while` loop needed? Won't buy now only have single item and quantity field?

Comment (ChetanGN) [reply templates/rzp-pdp-checkout-btn.php:26]: @varun-315 Some themes will not have the qty info on the 0th index so we need to loop it through to get the right qty selected.

**Review by ChetanGN — APPROVED** [review]
LGTM

**Comment by ChetanGN** [comment]
@poojarazor I hope you had tested these changes with other themes also. It should not break the flow for other themes.

**Pull request merged**